lib/ansible/modules/network/cnos/cnos_portchannel.py

```diff
@@ -378,34 +378,34 @@
   interfaceArg2: "port"
   interfaceArg3: "anil"
 
-- name: Test Port Channel - bfd
-  cnos_portchannel:
-  host: "{{ inventory_hostname }}"
-  username: "{{ hostvars[inventory_hostname]['ansible_ssh_user'] }}"
-  password: "{{ hostvars[inventory_hostname]['ansible_ssh_pass'] }}"
-  deviceType: "{{ hostvars[inventory_hostname]['deviceType'] }}"
-  outputfile: "./results/test_portchannel_{{ inventory_hostname }}_output.txt"
-  interfaceRange: 33
-  interfaceArg1: "bfd"
-  interfaceArg2: "interval"
-  interfaceArg3: 55
-  interfaceArg4: 55
-  interfaceArg5: 33
+#- name: Test Port Channel - bfd
```

Comment:
I really don't see the point of comments inside an EXAMPLES string...

Reply:
I will remove it now.